### PR TITLE
VACMS-8257: Adjust content admin role permissions

### LINKED ISSUE #8257: Content admin can't create media
## Describe the defect
Content admin can't create media

They used to be able to when they had "administer media", but that permission is too broad and includes some media config. 

They need more granular media permissions

## To Reproduce
1. As content admin (eg `beth.potts@va.gov`
2. Go to /node/add/vet_center
3. See that you can't add a Facility photo

## Expected behavior
You should be able to add a facility photo

I see a few other issues in the permissions. 

## ACs

<details><summary>All these permissions should be checked</summary>
Permissions_filtered_by_modules___VA_gov_CMS

</details>

### CMS Team
Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by netboss**
- Updated various user roles "Media" related permissions.
- Updated various user roles permissions in PHPUnit test: /tests/phpunit/Security/RolesPermissionsTest.php.

## Description

closes #8257 

## Testing done
Behat, PHPUnit and visual

## Screenshots
VACMS-8257-content-admin-add-media

## QA steps

As user content admin (eg beth.potts@va.gov)

1. Go to `/node/add/vet_center` and:
    - [ ] Verify and confirm you can add a Facility photo

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide CMS Team`
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS Experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product Support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/user.role.content_admin.yml [hunk 1]
```diff
@@ -5,8 +5,10 @@ dependencies:
   config:
     - filter.format.rich_text
     - filter.format.rich_text_limited
-    - node.type.banner
-    - node.type.promo_banner
```
Comment (swirtSJW) [inline config/sync/user.role.content_admin.yml:9]: These removals are not covered by the ACs and seem unexpected to me.

File: config/sync/user.role.content_admin.yml [hunk 2]
```diff
@@ -37,7 +39,6 @@ dependencies:
     - toolbar_menu
     - va_gov_backend
     - va_gov_build_trigger
-    - va_gov_menu_access
```
Comment (swirtSJW) [inline config/sync/user.role.content_admin.yml:40]: This removal is not covered by the ACs and is unexpected.

File: config/sync/user.role.content_editor.yml [hunk 3]
```diff
@@ -58,6 +58,7 @@ dependencies:
     - node.type.vet_center_locations_list
     - node.type.vet_center_mobile_vet_center
     - node.type.vet_center_outstation
+    - node.type.vha_facility_nonclinical_service
```
Comment (swirtSJW) [inline config/sync/user.role.content_editor.yml:61]: This addition is not covered by the ACs and is unexpected.

File: config/sync/user.role.content_publisher.yml [hunk 4]
```diff
@@ -47,14 +47,18 @@ dependencies:
     - node.type.support_service
     - node.type.va_form
     - node.type.vamc_operating_status_and_alerts
+    - node.type.vamc_system_billing_insurance
+    - node.type.vamc_system_medical_records_offi
     - node.type.vamc_system_policies_page
+    - node.type.vamc_system_register_for_care
```
Comment (swirtSJW) [inline config/sync/user.role.content_publisher.yml:53]: These additions are not covered by the ACs and is unexpected.

File: config/sync/user.role.content_publisher.yml [hunk 5]
```diff
@@ -47,14 +47,18 @@ dependencies:
...
     - node.type.vba_facility
     - node.type.vet_center
     - node.type.vet_center_cap
     - node.type.vet_center_facility_health_servi
     - node.type.vet_center_locations_list
     - node.type.vet_center_mobile_vet_center
     - node.type.vet_center_outstation
+    - node.type.vha_facility_nonclinical_service
```
Comment (swirtSJW) [inline config/sync/user.role.content_publisher.yml:61]: This addition is not covered by the ACs and is unexpected.

File: config/sync/user.role.content_reviewer.yml [hunk 6]
```diff
@@ -47,14 +47,18 @@ dependencies:
     - node.type.support_service
     - node.type.va_form
     - node.type.vamc_operating_status_and_alerts
+    - node.type.vamc_system_billing_insurance
+    - node.type.vamc_system_medical_records_offi
     - node.type.vamc_system_policies_page
+    - node.type.vamc_system_register_for_care
```
Comment (swirtSJW) [inline config/sync/user.role.content_reviewer.yml:53]: These additions is not covered by the ACs and is unexpected.

File: config/sync/user.role.content_reviewer.yml [hunk 7]
```diff
@@ -47,14 +47,18 @@ dependencies:
...
     - node.type.vba_facility
     - node.type.vet_center
     - node.type.vet_center_cap
     - node.type.vet_center_facility_health_servi
     - node.type.vet_center_locations_list
     - node.type.vet_center_mobile_vet_center
     - node.type.vet_center_outstation
+    - node.type.vha_facility_nonclinical_service
```
Comment (swirtSJW) [inline config/sync/user.role.content_reviewer.yml:61]: This addition is not covered by the ACs and is unexpected.

File: config/sync/user.role.vamc_content_creator.yml [hunk 8]
```diff
@@ -11,6 +11,7 @@ dependencies:
     - node.type.person_profile
     - node.type.press_release
     - node.type.regional_health_care_service_des
+    - node.type.vha_facility_nonclinical_service
```
Comment (swirtSJW) [inline config/sync/user.role.vamc_content_creator.yml:14]: This addition is not covered by the ACs and is unexpected.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
@netboss  There are a lot of changes here that are not covered by the issue or the testing steps.  Were there AC's added that need to be noted somewhere?

File: config/sync/user.role.content_admin.yml (hunk 1 above)
Comment (netboss) [reply config/sync/user.role.content_admin.yml]: @swirtSJW 
Restored those two configurations

File: config/sync/user.role.content_admin.yml (hunk 2 above)
Comment (netboss) [reply config/sync/user.role.content_admin.yml]: @swirtSJW 
Restored the `va_gov_menu_access` configuration

File: config/sync/user.role.content_publisher.yml (hunk 4 above)
Comment (netboss) [reply config/sync/user.role.content_publisher.yml]: @swirtSJW
Removed those three configurations

File: config/sync/user.role.content_publisher.yml (hunk 5 above)
Comment (netboss) [reply config/sync/user.role.content_publisher.yml]: @swirtSJW
Removed the `node.type.vha_facility_nonclinical_service` configuration

File: config/sync/user.role.content_reviewer.yml (hunk 6 above)
Comment (netboss) [reply config/sync/user.role.content_reviewer.yml]: @swirtSJW
Removed those three configurations

File: config/sync/user.role.content_reviewer.yml (hunk 7 above)
Comment (netboss) [reply config/sync/user.role.content_reviewer.yml]: @swirtSJW
Removed the `node.type.vha_facility_nonclinical_service` configuration

File: config/sync/user.role.vamc_content_creator.yml (hunk 8 above)
Comment (netboss) [reply config/sync/user.role.vamc_content_creator.yml]: @swirtSJW
Removed the node.type.vha_facility_nonclinical_service configuration

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**